### PR TITLE
Merging latest dev changes into new release branch off main

### PR DESCRIPTION
**Pull request by jeffcullis**

### CODE REVIEW

File: erddap_utils.sh
Comment (sjbruce) [inline erddap_utils.sh:1]: Are the erddap_gendata_mv() and erddap_xml_concat() utilities no longer required because of the move towards datasets.d?

**Review by sjbruce — COMMENTED** [review]
In general things look good.

The CANDHIS datasets are present in this version of **datasets.xml** so the external source .XML files for those datasets are not required.

In addition, the external .XML files for the CANDHIS datasets contain their original ERDDAP generated dataset ids, whereas the versions in datasets.xml reflect the proper dataset id, further diminishing the use of those files.

**Comment by sjbruce** [comment]
Addressed conflicts in datasets.xml - addition of CANDHIS and BIO datasets caused the conflict, they shouldn't be an issue now

**Comment by jeffcullis** [comment]
Pulling the release branch on production and restarting ERDDAP appeared to work smoothly, so I've now merged release into main branch and pushed that up as well (after tagging and pushing the tag for the commit on main that had been previously running).

**Pull request merged**